eng/docs/api/assets/header.html

```diff
@@ -34,8 +34,10 @@
 }
 
 function currentPackage() {
-  // Filled in by Doxygen template
-  return "$projectname";
```

Comment:
Should we make this an if/else so that if we ever switch back to projects it will "just" work?

Reply:
Discussed offline and resolved